.github/workflows/build-test-release.yml

```diff
@@ -150,36 +150,70 @@ jobs:
         with:
           submodules: true
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Setup kubernetes cluster
+        id: kubernetes
+        run: |
+          ./k3s.sh
+          kubectl get no
+      - name: Setup Splunk-Operator at cluster-scope
+        run: |
+          wget -O splunk-operator.yaml https://github.com/splunk/splunk-operator/releases/download/1.0.5/splunk-operator-cluster.yaml
+          kubectl apply -f splunk-operator.yaml
+          sleep 30
+          kubectl get ns
+          kubectl get pods -A
       - name: Setup for testing
         run: |
           pip install git+https://github.com/pixelb/crudini
           mkdir test-results-${{ matrix.splunk-version }}
+          pip install https://download.splunk.com/misc/packaging-toolkit/splunk-packaging-toolkit-1.0.1.tar.gz
+          ./spl_generation.sh
       - name: Splunk Up
         run: |
-          export SPLUNK_APP_PACKAGE=./tests/e2e/addons/TA_fiction
-          export SPLUNK_ADDON=TA_fiction
-          export SPLUNK_APP_ID=TA_fiction
-          ls -l deps/build/addonfactory_test_matrix_splunk/splunk_matrix.conf
-          export SPLUNK_VERSION=$(crudini --get deps/build/addonfactory_test_matrix_splunk/splunk_matrix.conf ${{ matrix.splunk-version }} VERSION)
+          sudo apt-get install -y curl jq
+          export NAMESPACE_NAME=ta-fiction-external
+          export SPLUNK_ADDON=TA_fiction-0.0.0-dev.spl
+          export SPLUNK_VERSION=${{ matrix.splunk-version }}
+          export SC4S_INDEX_URL=$( curl -s https://api.github.com/repos/splunk/splunk-configurations-base-indexes/releases/latest \ | jq -r '.assets[] | select((.name | contains("spl")) and (.name | contains("search_head") | not) and (.name | contains("indexers") | not ) and (.name | contains("forwarders") | not)) | .browser_download_url ')
           echo $SPLUNK_VERSION
-          docker-compose -f "docker-compose-ci.yml" build
-          SPLUNK_PASSWORD=Chang3d! docker-compose -f docker-compose-ci.yml up -d splunk
-          sleep 90
+          eval "echo \"$(cat ./pytest_splunk_addon/k8s_manifests/splunk_standalone/namespace.yaml)\"" >> ./namespace_updated.yaml
+          eval "echo \"$(cat ./pytest_splunk_addon/k8s_manifests/splunk_standalone/splunk_standalone.yaml)\"" >> ./splunk_standalone_updated.yaml
+          kubectl apply -f ./namespace_updated.yaml
+          kubectl run nginx -n $NAMESPACE_NAME --image=nginx --port=80 --expose
+          kubectl wait pod nginx --for=condition=ready --timeout=300s -n $NAMESPACE_NAME
+          kubectl cp ./tests/src/$SPLUNK_ADDON nginx:/usr/share/nginx/html -n $NAMESPACE_NAME
+          echo "        - http://nginx//$SPLUNK_ADDON" >> ./splunk_standalone_updated.yaml
+          kubectl apply -f ./splunk_standalone_updated.yaml -n $NAMESPACE_NAME
+          sleep 30
+          until kubectl logs splunk-s1-standalone-0 -c splunk -n $NAMESPACE_NAME  | grep "Ansible playbook complete"; do sleep 1; done
```

Comment:
Do we need these 2 lines here? Isn't `until ...` is enough?

Reply:
Yes, because after requesting the creation of a kubernetes resource it won't get scheduled on node. It can take some fraction of seconds to schedule then it will be able to access the logs of the pod.